### PR TITLE
Style: fixing typos

### PR DESCRIPTION
**Pull request by evansantos**
This correct some typos I found while I was reading readme.md content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,14 +57,14 @@ git commit -m "Create comments migration"
 O mais interessante ainda, seria fazer este commit após a criação da migration e do model. Assim, unificamos um commit para criação tanto do database, quanto do model:
 
 ```
-git commit -m "Create comments migration and Comment model"
+git commit -m "Create comments migration and comment model"
```
Comment (felipeorlando) [inline README.md]: In this case, we reffering to `Comment` as a model. 😄

**Review by felipeorlando — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (evansantos) [reply README.md]: Okay tho', I'll fix it :D

Comment (felipeorlando) [reply README.md]: luv u, beach ❤️

**Review by felipeorlando — APPROVED** [review]

**Comment by evansantos** [comment]
👍 👍

**Pull request merged**